.github/workflows/pytest.yml

```diff
@@ -0,0 +1,17 @@
+name: PyTest
+on: pull_request
+jobs:
+ pytest:
+ name: Run the application's tests with PyTest
+ runs-on: ubuntu-20.04
+ container: fedora:32
+ steps:
+ - name: Install Pipenv and Git
+ run: dnf install -y pipenv git
+ - name: Checkout code
+ uses: actions/checkout@v2
+ - name: Setup environment
+ run: pipenv sync --dev
+ - name: Run PyTest
+ run: pipenv run pytest -v
+ 
```

Comment:
Now you have an extra line, you shouldn't see here the red symbol

Reply:
I saw that and immediately modified the file, but I can see the change in the File Changed section

Reply:
I saw that and immediately modified the file.